### DISCUSSION TITLE
ABF vs WTMetaD PMF

### DISCUSSION THREAD

**Discussion by groponp** [Colvars/colvars#717] [Q&A] [answered]
Dear, I am performing the PMF calculation using the ABF method (classic) and tempered metadynamics, but the PMF I obtain are totally different. I am using the distance between monomers of my protein as a collective variable. When I did a test using each alanine, the PMF curves were very similar, but now when I use it for my system (protein dimer), the maximum values ​​of my curve with ABF reach up to 200 kcal/mol while with metadynamics it reaches only up to 40 kcal/mol (more reasonable), reading a little here and some articles, I couldn't find a clear reason for what I'm observing, maybe you can help me direct these results.

colvar common to both methods. 
```bash 
colvarsTrajFrequency      500   
colvarsRestartFrequency   500 

colvar {
    name AtomDistance
    width 0.10                 # Mesmo do que CHARMM-GUI and também BFEE2. Não considere lower/upperWall.
    lowerboundary 53.6         # O valor é zero, porque a diferencia de distanciaentre os dímeros é 0.0. 
    upperboundary 103.6        # O valor máximo para evaluar o PMF. 
    expandboundaries  on   


    distance {
        forceNoPBC       yes  
        group1 {
                atomsFile  P1.pdb    # PROA ou PROC. 
                atomsCol B 
                atomsColValue 1.0 
        }

        group2 { 
                atomsFile  P2.pdb  # PROB.
                atomsCol B 
                atomsColValue 1.0  
        }

    }
}

harmonicWalls {
    name ABF-Walls
    colvars AtomDistance
    lowerWalls 53.6          # lowerboundary. 
    upperWalls 103.6         # upperboundary.
    lowerWallConstant 10.0  # Força mesma como em charmm-gui. 
    upperWallConstant 10.0  # Força mesma como em charmm-gui. 


}
´´´
setting for ABF

```bash

abf {
    name     ABF-Distance
    colvars AtomDistance
    fullSamples 10000      
    hideJacobian           # Exclui a entropia geométrica. so para distance e distanceXY. 
                           # fonte: https://github.com/Colvars/colvars/issues/367
    historyfreq  500       # Salva o free energy gradient, sampling histograma e PMF no mesmo arquivo. 

}
´´´

setting for metadynamics 
```bash
metadynamics {
    name     WTMetaD-Distance
    colvars  AtomDistance 
    hillWeight              0.25     
    newHillFrequency        500   # Valor padrão.    
    hillwidth               0.5   
    writeHillsTrajectory    on     # Escreve un arquivo para armazenar os Hills (kcal/mol)
    wellTempered            on     # Favorece WellTempered Metadynamics.
    biasTemperature         4340   #  O Bias factor é calculado como:
                                   # biasFactor = (TemperatureMD + biasTemperature) / TemperatureMD
                                   # Onde TemperatureMD é a temperatura escolhida da simulação.
                                   # e biasTemperature é a temperatura escolhida aquí, na metadinâmica, biasFactor 15 cá. 
} 
´´´

**Comment by jhenin** [comment, marked as answer]
Hi, this sounds like a convergence issue, highly system-specific. To diagnose it I would compare the colvar trajectories for the two methods, and then the molecular trajectories in general. Something must be behaving qualitatively differently.

Side note: the hideJacobian option gives an ABF-specific type of free energy profile that is not equal to that given by metadynamics. I doubt this is the source of error here. In doubt, don't use hideJacobian.

**Comment by groponp** [comment]
Dear Porf @jhenin , I have been check out my trajectories, and i found that the unbinding process is different in both methods. In case of metadynamics Is there any trick to improve the convergence?.  Thanks so much in advanced.
